### PR TITLE
Upgrade node 14

### PR DESCRIPTION
**Pull request by nellshamrell**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -50,6 +50,7 @@
     "js-yaml": "^3.13.1",
     "linebyline": "^1.3.0",
     "lodash": "^4.17.21",
+    "lzma-native": "^8.0.1",
```
Comment (disulliv) [inline package.json:53]: Was this intentionally added? What's it used for?

Comment (nellshamrell) [reply package.json:53]: I believe that, in the Node 12 versions of our dependencies, this one was included in one of them. However, with the Node 14 versions of dependencies, it needs to be included on its own.

When I upgraded to Node 14 and ran the tests without this dependency, I got errors along the lines of:

```
node cannot find module lzma_native.node
```

When I install lzma-native as its own dependency, the tests ran as expected.

Comment (disulliv) [reply package.json:53]: Gotcha, looks fine then!

**Review by disulliv — APPROVED** [review]

**Comment by nellshamrell** [comment]
Planning on doing some more testing, then deploying this on Mon 12/6

**Comment by nellshamrell** [comment]
Deployed to dev.clearlydefined.io and am doing some testing:

- [ ] Harvest github component
- [ ] Harvest gitlab component
- [ ] Harvest maven central component
- [ ] Harvest maven google component
- [ ] Harvest crate component
- [ ] Harvest npm component
- [ ] Harvest nuget component
- [ ] Harvest pod component
- [ ] Harvest composer component
- [ ] Harvest gem component
- [ ] Harvest deb component

**Pull request merged**